keel-web/config/keel.yml

```diff
@@ -70,6 +70,13 @@ okHttpClient:
   readTimeoutMs: 59000
   interceptor.skipHeaderCheck: true
 
+resilience4j.retry:
+  configs:
+    default:
```

Comment:
Not a blocker, but you might want to define specific retry properties for each service and wire those into the respective classes. Can be done in a future PR.